Included contributing section into README:

fsenf opened the pull request:
Based on the discussions in PR #10 & issue #8, I added a small description to our README containing:
(1) requirements for PRs
(2) example data upload

fsenf committed:
included contributing section into README:
(1) requirements for PRs
(2) example data upload

JuliaKukulies commented:
Great! Maybe the PR requirements could also be in form of a [pull_request_template.md](https://github.com/climate-processes/tobac/pull/120), to be consistent with the other repo and that these checks will automatically appear when submitting a PR?

JuliaKukulies commented:
Great! Maybe the PR requirements could also be in form of a [pull_request_template.md](https://github.com/climate-processes/tobac/pull/120), to be consistent with the other repo and that these checks will automatically appear when submitting a PR?

fsenf committed:
added a pull request template that lists a set of requirements

fsenf commented:
@JuliaKukulies: Nice idea! I added a PR template based on your link!

fsenf commented:
@JuliaKukulies : Would you like to proceed the review?

JuliaKukulies commented on README.md:
```diff
@@ -13,3 +13,38 @@ A step-by-step description to run the tutorials with jupyter is located [here](d
+* **with your suggestions to change the documentation elements**, e.g. `index.rst`
+  * please care that sphinx-based rendering is still working correctly
+
+* **or with combination of both**
```
I think this line would slightly confuse me, should there be separate pull requests or is it OK to have the addition of the notebook and changes to `index.rst` in one?

JuliaKukulies commented on README.md:
```diff
@@ -13,3 +13,38 @@ A step-by-step description to run the tutorials with jupyter is located [here](d
+  * please check if your notebook is running in the recommended standard conda enviroment [see here]((docs/RunningTutorials.md) and update the [list of required python packages](./requirements.txt) if needed
+
+* **with your suggestions to change the documentation elements**, e.g. `index.rst`
+  * please care that sphinx-based rendering is still working correctly
```
Maybe provide a link on how to check this or in general how jupyter notebook works with Sphinx (e.g. https://docs.readthedocs.io/en/stable/guides/jupyter.html) ?

JuliaKukulies reviewed (approved):
Hey @fsenf, two small comments from me (see below). Otherwise it looks good, also with the PR template. Thanks for adding this one! I hope it is not an overkill of instructions now, but I think it is useful to have this PR check list to facilitate the contribution and review.

fsenf committed:
updated README text on contributing and added a documentation md for sphinx-based rendering of our tutorials

fsenf commented:
@JuliaKukulies : Thanks for your suggestions! I removed the bullet that confused you & I added a small docu on how to run sphinx-based rendering locally. Please have a look ...

fsenf commented:
@JuliaKukulies : PR ready to merge? What do you think?

JuliaKukulies commented:
Sorry for delaying this @fsenf, but I am happy with it now and would say go ahead and merge!

fsenf commented:
@JuliaKukulies : Great, and be convinced that I am always slower than you ;-)
